gapic-generator/templates/default/service/rest/test/method/_setup.erb

```diff
@@ -0,0 +1,38 @@
+class ClientStub
```

Comment:
I'm not too worried about this being repeated in every test. It's not that big, and this is generated code, and it exists only in the tests.

If it's still a concern for you, an option is to put it in the `helper.rb`. Maybe under a global namespace like `Gapic::Rest::TestClientStub`. I'd include a conditional that generates it only if the library contains rest services.